src/cmdstan/command.hpp

```diff
@@ -872,16 +880,16 @@ namespace stan {
                                   rng_t>
             cmd_advi(model,
                      cont_params,
+                     base_rng,
                      grad_samples,
                      elbo_samples,
-                     eta,
-                     base_rng,
                      eval_elbo,
                      output_samples,
                      &std::cout,
                      output_stream,
                      diagnostic_stream);
-          cmd_advi.run(tol_rel_obj, max_iterations);
+          cmd_advi.run(eta, adapt_engaged, adapt_iterations,
+            tol_rel_obj, max_iterations);
```

Comment:
This isn't critical, but here and on line 861, it's conventional to start the next line with the arguments under the last line.  cpplint's OK with your way, but it's easier to see the function if you have the room to indent.